sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2072,6 +2072,451 @@ private Response<RecognizeLinkedEntitiesResultCollection> RecognizeLinkedEntitie
+        //        string location = response.Headers.OperationLocation;
+
+        //        return new AnalyzeOperation(_serviceRestClient, _clientDiagnostics, location);
+        //    }
+        //    catch (Exception e)
+        //    {
+        //        scope.Failed(e);
+        //        throw;
+        //    }
+        //}
+
+        //private async Task<AnalyzeOperation> StartAnalyzeOperationAsync(MultiLanguageBatchInput batchInput, AnalyzeOperationOptions options, CancellationToken cancellationToken = default)
+        //{
+        //    options ??= new AnalyzeOperationOptions();
+
+        //    AnalyzeBatchInput analyzeDocumentInputs = new AnalyzeBatchInput(batchInput, new JobManifestTasks());
+
+        //    using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TextAnalyticsClient)}.{nameof(StartAnalyzeOperation)}");
+        //    scope.Start();
+
+        //    try
+        //    {
+        //        ResponseWithHeaders<TextAnalyticsAnalyzeHeaders> response = await _serviceRestClient.AnalyzeAsync(analyzeDocumentInputs, cancellationToken).ConfigureAwait(false);
+        //        string location = response.Headers.OperationLocation;
+
+        //        return new AnalyzeOperation(_serviceRestClient, _clientDiagnostics, location);
+        //    }
+        //    catch (Exception e)
+        //    {
+        //        scope.Failed(e);
+        //        throw;
+        //    }
+        //}
+
+        //#endregion
+
+        #region Health Async
+
+        /// <summary>
+        /// <a href="https://aka.ms/tanerpii"/>.
+        /// For a list of languages supported by this operation, see
+        /// <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/language-support"/>.
+        /// For document length limits, maximum batch size, and supported text encoding, see
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits"/>.
+        /// </summary>
+        /// <param name="document">The document to analyze.</param>
+        /// <param name="language">The language that the document is written in.</param>
+        /// <param name="options">The additional configurable <see cref="AnalyzeHealthOptions"/> </param>
+        /// <param name="cancellationToken"><see cref="CancellationToken"/></param>
+        /// <exception cref="RequestFailedException">Service returned a non-success
+        /// status code.</exception>
+        public virtual async Task<AnalyzeHealthOperation> StartAnalyzeHealthAsync(string document, string language = default, AnalyzeHealthOptions options = default, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(document, nameof(document));
+            options ??= new AnalyzeHealthOptions();
+
+            var documents = new List<string>() { document };
+
+            MultiLanguageBatchInput documentInputs = ConvertToMultiLanguageInputs(documents, language);
```

Comment:
instead of creating a list, you could just call `ConvertToMultiLanguageInput` i.e. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs#L429

That way you save the creation of an extra list